### ISSUE TITLE
Modules with same names are confusing to developers

### ISSUE THREAD

**Issue by osa1** [iu-parfunc/lvars#121]
We should probably rename some of the modules with names like:

- Internal.Control.LVish.SchedIdempotent
- Control.LVish.SchedIdempotentInternal

Users won't care since they're not supposed to be used by users, but it's confusing to developers.

**Comment by rrnewton** [comment]
Yes, that whole notion of putting "Internal" in front caused an ugly hiccup in the git history (caused merge problems after).

Further, the idea was just that it would get internal modules out of the way in the haddocks -- i.e. grouping them together rather than interspersing them.  But later I heard (from you?) that there's a simpler way to just hide those internal modules in the haddocks.

Besides `Foo.Bar.Internal`, with Internal trailing, is the standard idiom, for better or worse....

I can't even remember why there seem to be two modules that are both SchedIdempotent + Internal... that looks horrible.
